src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/GoogleDataprocDAO.scala

```diff
@@ -97,9 +98,9 @@ class GoogleDataprocDAO(protected val dataprocConfig: DataprocConfig, protected
   }
 
   // Using the given access token, look up the corresponding email of the user and get how long, in seconds, the token will expire in
-  def getEmailAndExpirationFromAccessToken(accessToken: String)(implicit executionContext: ExecutionContext): Future[(WorkbenchUserEmail, DateTime)] = {
+  def getEmailAndExpirationFromAccessToken(accessToken: String)(implicit executionContext: ExecutionContext): Future[(WorkbenchUserEmail, Instant)] = {
     val request = oauth2.tokeninfo().setAccessToken(accessToken)
-    executeGoogleRequestAsync(GoogleProject(""), "cookie auth", request).map{tokenInfo => (WorkbenchUserEmail(tokenInfo.getEmail), DateTime.now + tokenInfo.getExpiresIn.toInt)}
```

Comment:
The bug. `tokenInfo.getExpiresIn` is in seconds, but `DateTime.+` is in milliseconds.